.github/workflows/scripts/lighthouse/lighthouserc-auth.json

```diff
@@ -8,7 +8,9 @@
 				"non-composited-animations": "off",
 				"content-width": "off",
 				"csp-xss": "off",
-				"uses-passive-event-listeners": "off"
+				"uses-passive-event-listeners": "off",
+				"bf-cache": "warn",
+				"th-has-data-cells": "warn"
```

Comment:
Can we fix the issue in the generated code and keep it enabled as default?

Reply:
Hi @vishal423 

IMO, this is another issue,

the issue reported here (maybe `scope` should be `row` instead of `col`):

<img width="1414" alt="Screenshot 2023-04-16 at 20 26 40" src="https://user-images.githubusercontent.com/31362988/232309917-f6dfefea-2f20-496a-ad1e-4326e7a03bb7.png">

https://github.com/vishal423/jhipster-svelte-library/blob/e07f5422c18618bfcad8e290f2b946a0dc15ca8e/src/lib/table/table-header.svelte#L1-L3

please check the report here: 
https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1681646733659-18079.report.html

Reply:
`TableHeader` here according to `data-testId`: 

https://github.com/jhipster/generator-jhipster-svelte/blob/61b0d74a54ecc907f72d04bb490f17d3bf1d5ca0/generators/entity-client/templates/svelte/src/main/webapp/app/lib/entities/entity/entity-table.svelte.ejs#L34-L67

Reply:
Update:

Sorry @vishal423 , the reason is that there's no data showed in table:

<img width="1440" alt="Screenshot 2023-04-16 at 21 23 54" src="https://user-images.githubusercontent.com/31362988/232314439-84e3ad0b-ba95-45b2-96f7-dfae56598f5e.png">

here's an demobox you can test, comment/uncomment the lines`:
https://codesandbox.io/s/brave-pine-jbfuod?file=/src/App.tsx:277-483

<img width="1440" alt="Screenshot 2023-04-16 at 21 21 30" src="https://user-images.githubusercontent.com/31362988/232314396-85604250-6e06-4b45-a919-5d77c01b6edf.png">

Reply:
It can avoid the a11y issue using conditional table header rendering:

```ejs
<% if (containsRelationshipField) { %>
    <thead>
        ...
    </thead>
<% } %>
```

but I am not sure no data is expected here.

Reply:
I need to take a look. For now, lets keep it at the `warn` level